spec/acceptance/help_spec.rb

```diff
@@ -1,26 +1,47 @@
 require 'spec_helper_acceptance'
 
-describe "C98923 - Verify that 'puppet certregen --help' prints help text" do
-    describe command("puppet certregen --help") do
-      its(:stdout) { should match( /.*USAGE: puppet certregen <action>.*/ ) }
-      its(:stdout) { should match( /.*See 'puppet man certregen' or 'man puppet-certregen' for full help.*/ ) }
+describe "pupper help certregen" do
+  # NOTE: MODULES-4733 certregen is not currently compatible with ruby < 1.9
+  ruby_ver = 0
```

Comment:
These changes add a lot of logic that's performed in place; for the sake of maintainability these should be extracted into helper functions and placed in either a module or an RSpec shared context. This isn't a blocker but if this code is touched soon it should be refactored out.